pkgs/top-level/aliases.nix

```diff
@@ -1632,6 +1632,7 @@ mapAliases ({
   scrollkeeper = throw "'scrollkeeper' has been removed due to deprecated LibXML2 headers"; # Added 2022-11-08
   scyther = throw "scyther has been removed since it currently only supports Python 2, see https://github.com/cascremers/scyther/issues/20"; # Added 2021-10-07
   sdlmame = throw "'sdlmame' has been renamed to/replaced by 'mame'"; # Converted to throw 2023-09-10
+  searx = throw "'searx' has been removed as it is unmaintained. Please switch to searxng"; # Added 2023-10-03
```

Comment:
> 'searx' has been removed as it is unmaintained. Please switch to searxng

Can you put that in the commit message? I had to scroll a lot to find this info.